Source/Core/WinUpdater/WinUI.cpp

```diff
@@ -206,6 +205,14 @@ void SetDescription(const std::string& text)
 
 void MessageLoop()
 {
+  HRESULT result = CoInitializeEx(nullptr, COINIT_MULTITHREADED);
```

Comment:
This could potentially be named `coinit_result` or something more meaningful, but it's a small function anyway.